### ISSUE TITLE
fix: Jupyter Lab's widgets are not enabled. This is important to run exploratory data analysis.

### ISSUE THREAD

**Issue by portellaa** [ydataai/dockerfiles#37]
Include widgets as a lib to have installed in all the available bundles.

Enable widgets for Jupyter Lab https://ipywidgets.readthedocs.io/en/latest/user_install.html#installing-the-jupyterlab-extension

Thanks @fabclmnt 

@rafaelcpalmeida @victoriavilasb this should be done before the other images. 
Check between you who can do it. 
This was already done in the past in the jupyterlab layer, check the history if you want.
